content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+How to change the shipping label size
+=====================================
+
+Overview
+========
+
+With Odoo, there are a variety of different types of shipping  labels that can be selected for
+delivery orders. Depending on the types of shipping packages used, different label sizes may be more
+appropriate, and can be easily configured.
+
+Configuration
+=============
+
+In the :guilabel:`Inventory` module, go to :menuselection:`Configuration --> Delivery --> Shipping
+Methods.` Choose a delivery method and then click :guilabel:`Edit`. For the following example,
+*FedEx International* will be used, so select a delivery method and then click on :guilabel:`Edit`.
```

Comment:
This is an awkward sentence - maybe just stop it after *Fedex International*? Totally up to you